ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -121,32 +130,27 @@ objects:
 
         trap 'rc=$?; CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi; if test "${rc}" -ne 0; then touch /tmp/shared/exit; fi; exit "${rc}"' EXIT
 
-        # hack for bazel
```

Comment:
I think they would still need this bit, though.

Reply:
Not really, I took this from https://github.com/openshift/release/pull/11520

Reply:
Only to make the template work, that PR was not merged when I started :)